selfdrive/athena/athenad.py

```diff
@@ -472,6 +474,33 @@ def log_handler(end_event):
       cloudlog.exception("athena.log_handler.exception")
 
 
+def stat_handler(end_event):
+  while not end_event.is_set():
+    last_scan = 0
+    try:
+      curr_scan = sec_since_boot()
+      if curr_scan - last_scan > 10:
```

Comment:
This needs some kind of sleep to not use 100% CPU. Don't sleep too long or it will delay shutdown.